### ISSUE TITLE
Issue with "PeriodicWave": raw Math syntax on page

### ISSUE THREAD

**Issue by foolip** [mdn/content#3435]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/PeriodicWave

#### What information was incorrect, unhelpful, or incomplete?

"\left(a+bi\right)e^{i} , \left(c+di\right)e^{2i} , \left(f+gi\right)e^{3i}" appears on this page, which is unreadable. It looks a bit like LaTeX, presumably this was once turned into a math image?

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/API/PeriodicWave#example

#### What did you expect to see?

No expectations, just not this :)

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/periodicwave`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/PeriodicWave
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/periodicwave/index.html
* Last commit: https://github.com/mdn/content/commit/babc0473793bbe3cfa66cd96fae7f84348be373b
* Document last modified: 2021-02-08T10:05:46.000Z

</details>

**Comment by Rumyra** [comment]
Is this where the fourier equation is under the 'Example' heading @foolip ?

I ask because I see a rendered equation and not what you have above...

**Comment by foolip** [comment]
Yes, it's what follows "The coefficients of the Fourier transform should be given in ascending order".

It looks like it's MathML, so the issue is actually one of browser support?

**Comment by Rumyra** [comment]
Yes looks like it to me https://caniuse.com/mathml

It's not rendering in Opera or Edge for me which adds up. Wonder whether this is an issue for Yari as there are polyfills available 🤔

**Comment by teoli2003** [comment]
Yes, it displays correctly in Firefox. If I remember well, long time ago, there was on MDN a fallback from MathML to MathJax or similar. @elchi3 will surely know if it survived the move to yari, or if it is planned for the future. If not, we should find a solution, There is not much MathML on MDN – besides the MathML documentation that is – but we should track and fix it if MathJax is gone for good.

**Comment by Elchi3** [comment]
Yeah, this came up recently on other pages as well and @schalkneethling said that MathML polyfills didn't make it to yari. I'm not sure what the plan is. I guess someone needs to investigate the need for MathML or what we could do instead.

**Comment by peterbe** [comment]
Oh no! Did we lose this in Yari?

For JS features, we use https://polyfill.io/v3/url-builder/ so that it dynamically injects one of their polyfills. 
Is there a similar thing we can do for MathML? I'm quite ignorant of how MathML works in browsers. But it seems very possible to just extend the existing JS polyfill with one more specifically for MathML.

**Comment by Rumyra** [comment]
I just did a quick search just in the web directory and it came back with 253 `<math` matches in 86 files. Enough to be noticeable, but also not a huge percentage.

I also wonder what the strategy is for MathML with the move to markdown - does that influence how this is tackled?

**Comment by hamishwillee** [comment]
@wbamberg see ^^^ https://github.com/mdn/content/issues/3435#issuecomment-850348200

Mostly when I see mathml or mathjax in markdown it is marked up by $ guards and subsequently follows its own standard rules. I'm pretty sure github doesn't support it.

**Comment by wbamberg** [comment]
> I just did a quick search just in the web directory and it came back with 253 `<math` matches in 86 files. Enough to be noticeable, but also not a huge percentage.
> 
> I also wonder what the strategy is for MathML with the move to markdown - does that influence how this is tackled?

The plan is to keep it, so anything in `<math>` stays as HTML.

**Comment by hinell** [comment]
Please add also the following:
* [Math.cbrt()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/cbrt)
 * [Math.sin()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sin)
 * [Math.cos ()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/cos )
 * [Math.tan()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/tan)
 * [Math.asin()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/asin)
 * [Math.acos ()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/acos )
 * [Math.atan()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan)
 * [Math.asinh()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/asinh)
 * [Math.acosh()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/acosh)
 * [Math.atanh()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atanh)

**Comment by wbamberg** [comment]
Should we transfer this to mdn/yari?
